verticaltabs.js

```diff
@@ -843,6 +843,25 @@ VerticalTabs.prototype = {
     }
   },
 
+  delayResizeTabs: function (delay) {
+    if (delay <= 0) {
+      delay = 0;
+    }
+
+    if (this.resizeTimeout >= 0) {
```

Comment:
Hmm, I normally prefer to just check `if (this.resizeTimeout)`, because it's shorter, and because `setTimeout` returns a [a numeric, non-zero value](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout#Return_value) (probably largely to make Javascript developer's lives easier 😉).

Oh, also, we should make this if's test match the if test in `cancelResizeTabs` below.  (Either checking for `> 0`, or using `0` as the no-timeout value, and just checking `this.resizeTimeout`.